cmd/nodecmd/stop.go

```diff
@@ -33,8 +35,10 @@ The node stop command stops a running node in cloud server
 Note that a stopped node may still incur cloud server storage fees.`,
 		SilenceUsage: true,
 		Args:         cobra.ExactArgs(1),
-		RunE:         stopNode,
+		RunE:         stopNodes,
 	}
+	cmd.Flags().BoolVar(&authorizeAccess, "authorize-access", false, "authorize CLI to release cloud resources")
```

Comment:
we don't need this when executed from cli

Reply:
we need